addon/components/hyper-table/index.hbs

```diff
@@ -1,125 +1,129 @@
 <div class="hypertable__upper-header">
-  <div class="left-side">
-    {{#if this.manager.options.features.selection}}
-      <div class="selected-count">
-        {{format-number this._selectedCount}}
-      </div>
-    {{/if}}
+  <div class="fx-row">
+    <div class="left-side">
+      {{#if this.manager.options.features.selection}}
+        <div class="selected-count">
+          {{format-number this._selectedCount}}
+        </div>
+      {{/if}}
 
-    {{#if this.manager.options.features.search}}
-      {{#if this.searchComponent}}
-        {{#if (eq this._searchComponentType "string")}}
-          {{component this.searchComponent manager=this.manager}}
+      {{#if this.manager.options.features.search}}
+        {{#if this.searchComponent}}
+          {{#if (eq this._searchComponentType "string")}}
+            {{component this.searchComponent manager=this.manager}}
+          {{else}}
+            {{this.searchComponent}}
+          {{/if}}
         {{else}}
-          {{this.searchComponent}}
+          <Input
+            @type="text" @placeholder={{this.placeholder}} @value={{this._searchQuery}}
+            data-control-name="table_search_input" class="form-control upf-input" />
```

Comment:
Not sure what really changed at this level, but while you're in there, maybe you can replace the default Ember input with ours ?
```
<OSS::InputContainer @placeholder={{this.placeholder}} @value={{this._searchQuery}} data-control-name="table_search_input" />
```

Unless we don't have access to OSS here 🤔 I can't remember 🤷

Reply:
Yes we have access to OSS here, it was legacy code, I can try to update it :smile: